source/_components/sensor.transport_nsw.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: "Transport NSW"
+description: "Instructions on how to integrate timetable data for Transport NSW (Australia) within Home Assistant."
+date: 2018-08-18 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Transport
+ha_iot_class: "Cloud Polling"
+ha_release: 0.76
+---
+
+
+The `transport_nsw` sensor will give you the time until the next departure from a Transport NSW stop (bus, train or ferry).
+
+Get your free API key from [Transport NSW](https://opendata.transport.nsw.gov.au/).
+
+In order to find the stop id, just go to Google maps and click on the bus/train/ferry stop. It will give you there the stop ID.
+
+You can define a bus line, but if you don’t do it, the sensor will pick up the next stop event from any line servicing this stop.
+
+Then add the data to your `configuration.yaml` file as shown in the example:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: transport_nsw
+    name: 'Bus E80'
```

Comment:
> [This sample must contain only the required variables](https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration)

Reply:
Correct, like @WofWca reports.

We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.